satellite-software/src/LSM9DS1Sensor.cpp

```diff
@@ -349,4 +349,46 @@ void LSM9DS1Sensor::setGysroSensitivitySetting(GyroSensitivitySetting setting) {
 void LSM9DS1Sensor::setMagneticSensitivitySetting(MagneticSensitivitySetting setting) {
     eeprom_update_byte((uint8_t *)(getEEPROMBaseAddress()+OFFSET_MAGNETIC_SENSITIVITY), setting);
     _magneticSensitivity = setting;
+}
+
+uint8_t LSM9DS1Sensor::handleCommand(uint16_t cmdSequenceID, uint8_t command, uint8_t* recievedData, uint8_t recievedDataLen){
+    if(recievedDataLen != 1){
+        return CMD_STATUS_BAD_PARAM;
```

Comment:
This error should be `CMD_STATUS_BAD_DATA_LEN`

Reply:
Done